OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs

```diff
@@ -20,28 +22,37 @@ namespace OpenRA.Mods.Common.Scripting
 	[ScriptPropertyGroup("Ability")]
 	public class CaptureProperties : ScriptActorProperties
 	{
-		readonly CapturesInfo normalInfo;
+		readonly Captures[] capturesTraits;
```

Comment:
Variable `captures` is used below on https://github.com/MustaphaTR/OpenRA/blob/dd6495be728afb4e5bdc1c4a1941296c7c726049/OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs#L42

Are you sure this name should be changed? If so what should i rename `captures` at L42.

I did this naming similar to waht Infiltrates has.